src/components/Modals/Modal.vue

```diff
@@ -11,14 +11,15 @@
         :isOnboard="isOnboard"
         :semesterID="semesterID"
         :currentSemesters="currentSemesters"
+        :isCourseModelSelectingSemester="isCourseModelSelectingSemester"
         placeholderText='"CS 1110", "Multivariable Calculus", etc.'
         @duplicateSemester="disableButton"
         @close-current-model="closeCourseModal"
         @updateSemProps="updateSemProps"
         @toggle-left-button="toggleLeftButton"
         ref="modalBodyComponent"
-        :season="seasonCourse"
-        :year="yearCourse"
+        :season="season"
+        :year="year"
         :labels="true"
```

Comment:
For when you are opening the modal from the req bar, you can get rid of the labels by dynamically setting `:labels="false"`!
![image](https://user-images.githubusercontent.com/55263191/99920671-f63f8d80-2cf2-11eb-8a63-4136f582274c.png)

Reply:
I just routed isCourseModelSelectingSemester to hiding the labels, which is passed anyways. Works you mentioned